ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -44,7 +49,9 @@
  * the beginning until now, or any new messages since then), and writing to it.
  */
 
+// CHECKSTYLE_RULES.OFF: ClassDataAbstractionCoupling
```

Comment:
@hjafarpour has a PR coming that will split this class into `CommandStore` and `CommandTopic` which should fix this coupling issue for you. You may want to hold off committing until he has committed his...